src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="numbers-container">
```

Comment:
You calendar should have modifiers such as `start-day` and `month-length`. 
Working with those modifiers we can configure our calendar.
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--30">
```